source/channels/make-calls.rst

```diff
@@ -109,9 +123,16 @@ JS console logs
 Web app
 ^^^^^^^
 
-- **Chrome**: CMD+OPTION+J (macOS)/CTRL+SHIFT+J (Windows, Linux, ChromeOS)
-- **Firefox**: CMD+SHIFT+J (macOS)/CTRL+SHIFT+J (Windows, Linux, ChromeOS).
-- **Safari**: Enable Developer Menu in **Safari > Preferences > Advanced > Show Develop Menu in Menu Bar**. Then **Develop > Show Javascript Console**. Right-click on the console and select **Save to file** to download the logs.
++---------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Service                         | Ports                                                                                                                                                                                                                 | 
```

Comment:
I'm not sure what `Service` and `Ports` means here?

Reply:
Oh, good catch, thank you.